src/ansi.rs

```diff
@@ -1525,6 +1571,22 @@ where
                     }
                 }
             },
+            ('m', [b'>']) => {
+                let mode = match (next_param_or(1) == 4).then(|| next_param_or(0)) {
+                    Some(0) => ModifyOtherKeys::Reset,
+                    Some(1) => ModifyOtherKeys::EnableExceptWellDefined,
+                    Some(2) => ModifyOtherKeys::EnableAll,
+                    _ => return unhandled!(),
+                };
+                handler.set_modify_other_keys(mode);
+            },
+            ('m', [b'?']) => {
+                if params_iter.next() == Some(&[4]) {
+                    handler.report_modify_other_keys();
+                    return;
+                }
+                unhandled!();
```

Comment:
```suggestion
                if params_iter.next() == Some(&[4]) {
                    handler.report_modify_other_keys();
                } else {
                    unhandled!();
                }
```

Ton of control flow complexity in this file, probably better to avoid return.